content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -84,8 +84,10 @@ selected:
 - :guilabel:`Pass - Fail` checks specify a criterion that products must meet for the check to pass.
 - :guilabel:`Measure` checks prompt employees to record a measurement of the product that must be
   within a tolerance of a norm value for the check to pass.
-- :guilabel:`Worksheet` checks provide an interactive worksheet that must be filled out by the
-  employee processing the check.
+- :guilabel:`Worksheet` checks require the employee processing the check to fill out an interactive
+  worksheet.
```

Comment:
On line 68, I couldn’t find the field "Every #% of Transfers." Instead, I saw "Every #% of Operations," which seems to have been renamed between Odoo 17 and 18. I’d suggest updating that here for accuracy!

![image](https://github.com/user-attachments/assets/be2f5e45-3bf2-4a40-9c41-08bc1ce50e08)

Also, I found "Every #% of Transfers" a bit tricky to understand at first. Maybe consider adding a quick example to clarify? For instance: *Randomly check every 20% of operations means... [you can complete this idea!]*